src/components/Character.js

```diff
@@ -1,23 +1,84 @@
 import { createElement } from "../utils/elements";
+import "./Character.css";
+import favoritesEnabled from "../assets/favorites-enabled.svg";
+import favoritesDisabled from "../assets/favorites-disabled.svg";
 
-export function Character( {name, imgSrc}) {
-    
-    const fullName = createElement("h3", {
-        className: "character-fullName",
-        innerText: name,
-    });
-    
-    const image = createElement("img", {
-        className: "character-img",
-        src: imgSrc,
-        alt: name
-    })
-    
-    const container = createElement("div",{
-        className: "character-Container",
-        children: [fullName, image]
-    });
-
-    return container;
-
-};
\ No newline at end of file
+export function Character({ ...prop }) {
```

Comment:
Should be props because its multiple props, also why don't you destructure the props? Because there is a set amount of props your function can process.
```suggestion
export function Character({id, name, src, status, species, type, origin, location, type, isFavorite}) {
```

Reply:
i had that before, exactly like that, i wanted to test it out, my old / your way ist way more obvious